### ISSUE TITLE
Home: Voice Over

### ISSUE THREAD

**Issue by caphilomena** [sisalee1/Sisalee-s-Portfolio#18]
##Link
[Home](https://sisaleenolan.com/)

##Issue
For Voice Over the skip to main doesn't go to your text, it goes to your image which doesn't have alt text. 

##Possible Solution
Add alt text to your image to make it more understandable to someone listening.
